include/aspect/introspection.h

```diff
@@ -51,23 +56,33 @@ namespace aspect
    * @ingroup Simulator
    */
   template <int dim>
-  struct Introspection
+  struct Introspection: public dealii::FEVariableSystem<dim>
```

Comment:
I think of ``FEVariableSystem`` to be independent of ASPECT (and I will use this in other projects and in fact was using an older variant of this already), while ``Introspection`` is certainly ASPECT specific. That is a good enough reason for me to keep them separate.

I can move it out of the dealii namespace for now though.